app/models/game.rb

```diff
@@ -61,4 +62,18 @@ def location_is_under_attack_by_color?(color, row, col)
 
     false
   end
+
+  def forfeit!(user)
```

Comment:
Instead of `user`, could be super explicit here and use something like `forfeiting_user`. Then later in your method, it's super obvious which user lost.